extension/src/contentscript/player.ts

```diff
@@ -1,7 +1,8 @@
 import { Runtime } from "webextension-polyfill-ts";
+import { MessageType } from "@/types";
 
 export interface Event {
-    type: EventType;
+    eventType: EventType;
```

Comment:
c'est pas un peu redondant ? tu avais un autre type qqpart ?

Reply:
The `type` key is already used for the message type, when we send a message through the websocket

Reply:
What about VideoEventType then ? (and maybe eventually we'll rename the other Type enum to WebsocketEventType or something else)

Reply:
Done